x-pack/test/functional/apps/security/remote_cluster_security_roles.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { keyBy } from 'lodash';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+const EDIT_ROLES_PATH = 'security/roles/edit';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const browser = getService('browser');
+  const log = getService('log');
+  const security = getService('security');
+  const PageObjects = getPageObjects(['security', 'common', 'header', 'discover', 'settings']);
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Remote Cluster Privileges', function () {
+    const customRole = 'rc-custom-role';
+
+    before('initialize tests', async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/security/dlstest');
```

Comment:
Question: I cant remember if `esArchiver` flakiness was fixed or maybe the guidance changed on using `getService('kibanaServer').importExport.load()`

We can check during sync tomorrow.